public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+const { __ } = wp.i18n;
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { Edit } from './edit';
+
+export const NAME  = 'wordcamp/schedule';
+export const LABEL = __( 'Schedule', 'wordcamporg' );
+export const ICON  = 'schedule';
+
+const supports = {
+	align: [ 'wide', 'full' ],
+};
+
+export const SETTINGS = {
+	title       : __( 'Schedule', 'wordcamporg' ),
+	description : __( "Display your WordCamp's awesome schedule.", 'wordcamporg' ),
+	icon        : ICON,
+	category    : 'wordcamp',
+	supports    : supports,
+	edit        : Edit,
+	save        : () => null,
+
+	// todo add keywords? if yes, do for all blocks as separate commit, prob commit to master b/c this branch already huge
+
+	/*
+	 * Make the block full-width _only_ in the editor.
+	 *
+	 * It's very common for camps to have 3-4 tracks, but the editor's content area is very narrow, which makes it
+	 * difficult to read the schedule. Defining the block as full-width block gives it more space so it can be
+	 * easier to read, but we don't want to do that on the front-end, because that would unnecessarily add extra
+	 * styles that get in the way of organizer's customizing their CSS. It would also behave inconsistently between
+	 * themes that do and don't support the extra alignments that Gutenberg adds.
+	 *
+	 * Using the standard mechanism for full-width blocks would also add a toolbar icon which could be used to turn
+	 * off the wider view in the editor, which could cause confusion and clutter, without added benefit.
+	 *
+	 * Another potential solution would have been to manually add CSS rules to make the block full-width in the editor,
+	 * but that would have meant duplicating the existing rules from Core, and tightly coupling the editing experience,
+	 * which wouldn't be future-proof.
+	 *
+	 * Adding the alignment through `getEditWrapperProps()` allows us to achieve the desired result without any side-
+	 * effects.
+	 */
+	//getEditWrapperProps( props ) {
+	//	return {
+	//		'data-align': 'full',
+	//		// maybe apply this conditionally based on screen width? show responsive layout on smaller screen?
+	//	};
+    //},
+	// probably remove ^ in favor of `supports align wide full` above, but test with 1 and 2 column schedules once get live data integrated
+	// if do that, then maybe turn `full` on by default if the camp has 3+ tracks, or maybe just always, and let them turn it off if they want
+	// probably safe to always make it the default, as long as grid is center-aligned to content area, instead of left-aligned?
```

Comment:
It seems to me like something that should work before the block is merged. The switch from `getEditWrapperProps()` to `supports` has already been done, it just needs to be tested more before I'm confident that it's working in all cases.

I'm not planning to leave a bunch of `todo` comments in the code after the PR is merged, 99% of those are reminders for myself to fix/complete things while the branch is a WIP.

The reason I left those comments in the PWA files was because that feature was rushed to get deployed long before it was ready for production, and I wanted to make sure the code was brought up to standards before being activated for all sites. Having comments in the code means they're a lot less likely to get lost/ignored than if they're buried in some external issue tracker.

Reply:
I still think adding wide/full alignment support can be done in a following PR – get a basic schedule working in one PR, and then iterate on it. This isn't a live feature, it doesn't need to be perfect before the first PR is merged, and smaller, single-feature changes are easier for your teammates to review.